etl/experiments.kp/knowledge.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Experiment Job
+authors:
+- Frank Bertsch
+tags:
+- experiment
+- firefox
+created_at: 2017-02-01 00:00:00
+updated_at: 2017-02-07 14:16:47.616739
+tldr: 'We take all the pings from yesterday, get the information about any experiments:
+  those that started, those running, and those that ended. These are aggregated by
+  channel and outputted to files in s3.'
+---
+
+```python
+from datetime import datetime as dt, timedelta, date
+import moztelemetry
+from os import environ
+
+# get the desired target date from the environment, or run
+# on 'yesterday' by default.
+yesterday = dt.strftime(dt.utcnow() - timedelta(1), "%Y%m%d")
+target_date = environ.get('date', yesterday)
+```
+
+```python
+from moztelemetry.dataset import Dataset
+
+sample_rate = environ.get('sample', 1)
+pings = Dataset.from_source("telemetry") \
```

Comment:
You can use the "telemetry-experiments" dataset which has the following dimensions:
- submissionDate
- docType
- activeExperimentId

You'll need to filter on `appName == Firefox` later though.